ote/src/cljs/ote/app/controller/pre_notices.cljs

```diff
@@ -13,21 +13,25 @@
 (defn valid-notice? [notice]
   true)
 
-(declare ->LoadPreNoticesResponse ->LoadPreNoticesFailure)
+(tuck/define-event ServerError [response]
+  {}
+  (assoc app :flash-message-error (tr [:common-texts :server-error])))
+
+(declare ->LoadPreNoticesResponse)
 
 ;; Load the pre-notices that are available
 (tuck/define-event LoadOrganizationPreNotices []
   {:path [:pre-notices]}
   (comm/get! "pre-notices/list"
              {:on-success (tuck/send-async! ->LoadPreNoticesResponse)
-              :on-failure (tuck/send-async! ->LoadPreNoticesFailure)})
+              :on-failure (tuck/send-async! ->ServerError)})
   :loading)
 
 (tuck/define-event LoadAuthorityPreNotices []
```

Comment:
I would suggest that in this point we should move all elykeskus related stuff away from this controller and create a new one. So we can keep our controllers short and clean.